nireports/tests/test_reportlets.py

```diff
@@ -597,10 +596,7 @@ def test_create_cmap(outdir):
         key1 == key2
         for key1, key2 in zip(orig_cmap._segmentdata.keys(), ls_cmap._segmentdata.keys())
     ]
-    assert [
-        np.allclose(val1, val2)
-        for val1, val2 in zip(orig_cmap._segmentdata.values(), ls_cmap._segmentdata.values())
-    ]
+    assert list(map(np.allclose, orig_cmap._segmentdata.values(), ls_cmap._segmentdata.values()))
```

Comment:
This looks like a bad check. Probably should have been:

```suggestion
    assert all(map(np.allclose, orig_cmap._segmentdata.values(), ls_cmap._segmentdata.values()))
```